Makefile

```diff
@@ -3,6 +3,7 @@ ERLANG_BIN       = $(shell dirname $(shell which erl))
 GIT_VERSION      = $(shell git describe --tags)
 OVERLAY_VARS    ?=
 REBAR ?= $(BASE_DIR)/rebar3
+PSQL_MIGRATION_EXEC = ./psql_migration
```

Comment:
Make sure psql_migration is compiled and then used. 
DO NOT add already compiled psql_migration as it may cause issues on different hosts like M1.